src/ruuvitag_ble/parser.py

```diff
@@ -23,11 +33,15 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             return None
 
         data_format = raw_data[0]
-        if data_format != 0x05:
+        try:
+            decoder_cls: Type[Union[DataFormat3Decoder, DataFormat5Decoder]] = (
+                decoder_classes[data_format]
+            )
```

Comment:
A type checker should be able to find the type out from the dict's value type:
```suggestion
            decoder_cls = decoder_classes[data_format]
```
but if that doesn't work, let's at least be futuristic:
```suggestion
            decoder_cls: type[DataFormat3Decoder | DataFormat5Decoder] = (
                decoder_classes[data_format]
            )
```